<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WalletDialog</class>
 <widget class="QDialog" name="WalletDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>503</width>
    <height>332</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="sendPushButton">
   <property name="geometry">
    <rect>
     <x>392</x>
     <y>274</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
  </widget>
  <widget class="QPushButton" name="receivePushButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>274</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Receive</string>
   </property>
  </widget>
  <widget class="QLabel" name="numKeys">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>280</y>
     <width>233</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>12</y>
     <width>145</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Transactions</string>
   </property>
  </widget>
  <widget class="QLabel" name="numTransactions">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>290</y>
     <width>233</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="synchronizePushButton">
   <property name="geometry">
    <rect>
     <x>228</x>
     <y>274</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Synchronize</string>
   </property>
  </widget>
  <widget class="QTreeView" name="walletTableView">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>44</y>
     <width>471</width>
     <height>192</height>
    </rect>
   </property>
   <property name="indentation">
    <number>0</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
